test/e2e/cluster_upgrade.go

```diff
@@ -48,6 +48,11 @@ var _ = framework.KubeDescribe("Upgrade [Feature:Upgrade]", func() {
 			cm.Register(func(sem *chaosmonkey.Semaphore) {
 				// Close over f.
 				testServiceRemainsUp(f, sem)
+				testSecretsDuringUpgrade(f, sem)
```

Comment:
@quinton-hoole, had similar thoughts. ok lets keep all the upgrade tests under single upgrade as of now. I will change the "It()" clause to be generic.